src/rgw/rgw_sal_d4n.cc

```diff
@@ -0,0 +1,567 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
```

Comment:
@samarahu @dang should this file and it's header be moved into `driver/d4n`.

Reply:
Yep.